### ISSUE TITLE
Add a11y notice about prefix & suffix elements in components

### ISSUE THREAD

**Issue by rolfsmeds** [vaadin/docs#1902]
Prefix and suffix elements are generally not announced by screen readers. This should be mentioned for text-field derivatives and buttons.

(The prefix and suffix features are described in Text Field docs)

**Comment by web-padawan** [comment]
The [notice](https://github.com/vaadin/docs/blob/4a872be1f3d3ba4af012b13a4e26a8098f1a118f/articles/components/_input-field-common-features.adoc?plain=1#L78) was added in #2337 as part of adding "basic features" list to field components. Closing as fixed.
